run_test.py

```diff
@@ -62,7 +62,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda65',
-            'cudnn': 'cudnn2',
+            'cudnn': 'cudnn5',
```

Comment:
This change does not seems to be related to CuDNN 5.1. Should we include this change in this PR?